fcrepo-export/src/main/java/org/fcrepo/exporter/Exporter.java

```diff
@@ -147,7 +149,10 @@ void writeResponse(final FcrepoResponse response, final File file)
 
         final List<URI> describedby = response.getLinkHeaders("describedby");
         for (final Iterator<URI> it = describedby.iterator(); describedby != null && it.hasNext(); ) {
-            exportDescription(it.next());
+            exportDescription(response.getUrl(), it.next());
+            if (it.hasNext()) {
+                throw new RuntimeException("fcrepo does not produce multiple describedby headers.");
```

Comment:
+1 to avoiding the semantic ditch of discussing the deeper issue of what the binary's description really is — I think that's great work in the API spec context, but we want working code this week, so we need to stay focused.

I think it's worth considering if we should inspect the RDF and use the subject of the triples to serialize the RDF to disk.  I don't know what you would do if there were two or more descriptions of the same URI, though.  The original suggestion was to just add a suffix to make them unique, so you'd have `/rest/file1.ttl`, `/rest/file1-1.ttl`, `/rest/file1-2.ttl`, etc.  That avoid overwriting the other descriptions, but then I don't know how you'd match descriptions to files if some of them were updated but not others.  Or if some disappeared.

I decided to use the request URL to generate the file path, reasoning that if we retrieve new RDF from the same URL, then we'd want to replace that file on disk.  This seems like a good 1-to-1 mapping that makes it easy to translate back and forth.  We will have files named after fcr:metadata because that's the URL in the describedby Link header, but it's not baked-in in any way.